components/src/main/res/values/styles.xml

```diff
@@ -105,7 +105,7 @@
 
     <style name="TextInput" parent="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox">
         <item name="android:gravity">top|start</item>
-        <item name="android:textColorHint">@color/hmrc_dark_text</item>
+        <item name="android:textColorHint">@color/hmrc_grey_1</item>
```

Comment:
This will change the colour for dark mode too. Do we definitely want to do that as it's not mentioned in the ticket?

Reply:
@lojbrooks I have checked with @marcbelle and he is happy with the design in dark mode. I have updated the PR with before & after screenshots for light and dark mode, along with the colour contrast results.